diff --git "a/Git \346\214\207\344\273\244.txt" "b/Git \346\214\207\344\273\244.txt"
new file mode 100644
index 0000000..a1db23e
--- /dev/null
+++ "b/Git \346\214\207\344\273\244.txt"	
@@ -0,0 +1,47 @@
+Git 指令
+
+1. git版本查詢 
+
+git --version  
+
+2.git設定套件
+
+git config --list
+
+3.設定name
+
+git config --global user.name "?????"
+
+4.設定信箱用於連結github
+
+git config --global user.email "??????@gamil.com"
+
+5.移動到指定目錄
+
+cd C:
+
+6.退回上一層
+
+cd ..
+
+7.
+
+ls -al
+
+8.建立.git的隱藏資料夾(代表這個地方受到git控管)
+
+git init 
+
+9.全部加入控管
+
+git add .(點前面有空白) 
+
+10.commit 到下一個位置
+
+git commit -m "create aaa"
+
+11.修改完的檔案丟回暫存區
+
+git add . (點前面有空白) 
+
+12.
diff --git a/bbb.html b/bbb.html
index d155891..f016a14 100644
--- a/bbb.html
+++ b/bbb.html
@@ -1,2 +1 @@
 <!>
-ssssss
\ No newline at end of file
